src/static/riot/competitions/tile/competition_tile.tag

```diff
@@ -1,104 +1,137 @@
 <competition-tile>
-
-    <div class="ui grid">
-        <div class="ui middle aligned attached message stretched row main-wrapper">
-            <div class="four wide column">
-                <div class="ui square bordered small image">
-                    <img src="https://i.imgur.com/n2XUSxU.png">
-                    <!--<img src="{ logo }">-->
-                </div>
+    <a class="link-no-deco" href="./competitions/{id}">
+        <div class="tile-wrapper">
+            <div class="ui square tiny bordered image img-wrapper">
+                <img src="{logo}">
             </div>
-            <div class="nine wide column">
-                <div class="ui row">
-                    <div class="twelve wide column">
-                        <h1 class="ui large header">
-                            {title}
-                        </h1>
-                        <p class="comp-tile-paragraph">
-                            A competition description
-                            {description}
-                        </p>
-                    </div>
-                    <div class="four wide right justify left align column">
-                        <div align="right" class="content">
-                            <i>Organized by: <b>Someone</b></i>
-                        </div>
-                    </div>
-                </div>
-                <div class="ui row">
-                    <div class="sixteen wide column">
-                        <div class="content">
-                            <!--Tags?-->
-                        </div>
-                    </div>
-                </div>
-                <div class="ui row">
-                    <div class="sixteen wide column">
-                        <div class="content">
-                            Tags: <b>Beginner</b>, <b>AutoML</b>
-                            <br>
-                            Admins: <b>tthomas63</b>
-                        </div>
-                    </div>
-                </div>
+            <div class="comp-info">
+                <h4 class="heading">
+                    {title}
+                </h4>
+                <p class="comp-description">
+                    {description}
```

Comment:
Can eliminate looping through these descriptions on mount by doing something like this:
`{description.substring(0,90) + (description.length > 90 ? '...' : '')}`

Reply:
Very clean! Thanks for this!